EndlessClient/ControlSets/LoginPromptControlSet.cs

```diff
@@ -110,7 +110,7 @@ private IXNATextBox GetLoginUserNameTextBox()
                 _textBoxCursor)
             {
                 MaxChars = 16,
-                DefaultText = "Username",
+                DefaultText = "",
```

Comment:
You know, I _really_ like my default text, but the more I think about it it's kind of stupid lol